david-gabriel/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Lab 1</title>
+  <!-- TODO: Add a meta tag to properly set the viewport and initial scale -->
+  <meta name="viewport" content="width=device-width" />
```

Comment:
You need to add the `initial-scale` to your viewport tag to ensure that it works properly.

Reply:
Added and updated the PR!